### DISCUSSION TITLE
Error compiling/calculate

### DISCUSSION THREAD

**Discussion by ViniciusCezarB** [Anura3D/Anura3D_OpenSource#81] [Q&A] [answered]
Hi everyone, i'm having this problem to compile. But i maked a test and this problem emerged, don't have the file Anura3D_2023.exe and others files. Please help me.
![image](https://user-images.githubusercontent.com/84789119/225768087-d957e43d-4eef-4689-a056-7f737f79f918.png)
"the system cannot find the path specified"
![image](https://user-images.githubusercontent.com/84789119/225767556-232d38bb-a4ba-4ed9-b38d-0114ecd76f13.png)
![image](https://user-images.githubusercontent.com/84789119/225768222-9381865b-ec7d-4b8a-9ad8-caf497dd8eda.png)

**Comment by FrancescaCeccato** [comment, marked as answer]
My first suggestion is to symplify the path, do not use special characters and spaces. This may help.

**Reply by murphy-john** [reply]
@ViniciusCezarB, compiling the code in cloud drives like OneDrive, Box, GoogleDrive, etc. can cause issues. Try compling the code in a location that's not updating to the Cloud. 

Follow the guides in the tutorial manual and the YouTube videos for compiling the code and moving the executable to the directory.

**Reply by ViniciusCezarB** [reply]
Thank you, it worked. I removed Anura3D from the Cloud, and ran the VS debug.
